backend-dotnet/Meowmentum.Server.Dotnet.Api/Meowmentum.Server.Dotnet.Api/Controllers/AuthController.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Meowmentum.Server.Dotnet.Business.Abstractions;
+using Meowmentum.Server.Dotnet.Shared.Requests.Registration;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Meowmentum.Server.Dotnet.Api.Controllers;
+
+[Route("api/[controller]")]
+[ApiController]
+public class AuthController(IAuthService authService) : ControllerBase
+{
+    [HttpPost("register")]
+    [ProducesResponseType<string>(StatusCodes.Status200OK)]
+    [ProducesResponseType<string>(StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    public async Task<IActionResult> Register([FromBody] RegisterUserRequest request, CancellationToken token = default)
+    {
+        var result = await authService.RegisterUserAsync(request, token);
+
+        if (result.IsSuccess)
+            return Ok("User registered successfully. Please verify your email!");
```

Comment:
if you want to return message in 200 response, you should return it in result object from service(controllers should not contain any logic) and also add it to constants